plugins/adobe-analytics-compatible-version/index.ts

```diff
@@ -120,13 +121,19 @@ import { ValidationPluginResult } from 'types/validationPlugin';
     ? {
         events: [],
         message:
-          'The installed Assurance SDK and Analytics Edge Bridge Extensions are compatible.',
+          'The installed Assurance SDK and Analytics Edge Bridge extensions are compatible.',
         result: 'matched'
       }
+    : !analyticsVersion
```

Comment:
Do we want to do this check or just let it show the generic message 
'The installed Assurance SDK and Adobe Analytics extensions are compatible.',

While Analytics is not installed.

Reply:
Please add a scenario for when the extension is not installed. That would be awesome!

Reply:
Keeping !analyticsVersion check.